ads-service-fixed/Dockerfile

```diff
@@ -22,9 +22,12 @@ RUN pip install --no-cache-dir -r requirements.txt
 # Set default Flask app and development environment
 ENV FLASK_APP=ads.py
 
-# Listen on 5002
-EXPOSE 5002
+# Pass in Port mapping (default to 9292)
+ARG ADS_PORT=9292
```

Comment:
Is this preparing for a use case where we'd build images with a different default port? In other words, why not just `ENV ADS_PORT=9292` on line 29?